### DISCUSSION TITLE
A sign of the community

### DISCUSSION THREAD

**Discussion by LudicusX** [iota-community/Shimmer-Governance#18] [Ideas]
A sign of the community

The biggest fear of the active community is that they won't get their desired NFTs due to an early sell-out.
My proposal is to delet the limits of gold (1000 pieces), silver (3000 pieces), bronze (6000 pieces) to "limitless amounts". The period in which you can buy is e.g. limited to a day or a week.
Thus, each community member would get their desired NFTs and the final number of NFTs would actually reflect the current community before shimmer network comes to life.

If desired or technically possible, DID could ensure that the maximum number of NFTs for each member would be limited. But I don’t think it is necessary because the investment also represents a risk of loosing money. --> Check „Boost“ below
There would also be a lot more money going into the Shimmer / Community Fund.

"Boost"
The boost could be cut in half every year, two years or five years, that the second half will be donated to the fund to provide constant liquidity for future developments, community projects and research. So there is still a bit reward left as a thank for the constant liquidity that flows to the fund.

Advantages:
- Fair distribution to interested investors
- a lot more money would go into the fund
- the boost goes towards zero over time, so the risk of a centralization would be very low
- Constant liquidity in the fund

**Comment by antonionardella** [comment]
I see you posted this in the main thread. Is it ok for you if I close this one?

**Reply by LudicusX** [reply]
yes please - thanks :)
